exchange-v2/test/RaribleManager.test.js

```diff
@@ -0,0 +1,578 @@
+const RaribleTransferManagerTest = artifacts.require("RaribleTransferManagerTest.sol");
+const TransferProxyTest = artifacts.require("TransferProxyTest.sol");
+const TestRoyaltiesRegistry = artifacts.require("TestRoyaltiesRegistry.sol");
+const ERC20TransferProxyTest = artifacts.require("ERC20TransferProxyTest.sol");
+
+const TestERC20 = artifacts.require("TestERC20.sol");
+
+const TestERC721RoyaltiesV1 = artifacts.require("TestERC721RoyaltiesV1.sol");
+const TestERC721RoyaltiesV2 = artifacts.require("TestERC721RoyaltiesV2.sol");
+const TestERC1155RoyaltiesV2 = artifacts.require("TestERC1155RoyaltiesV2.sol");
+const TestERC1155RoyaltiesV1 = artifacts.require("TestERC1155RoyaltiesV1.sol");
+
+const ERC721_V1_Error = artifacts.require("TestERC721WithRoyaltiesV1_InterfaceError.sol");
+const ERC1155_V2_Error = artifacts.require("TestERC1155WithRoyaltiesV2_InterfaceError.sol");
+
+const ERC721LazyMintTest = artifacts.require("ERC721LazyMintTest.sol");
+const ERC1155LazyMintTest = artifacts.require("ERC1155LazyMintTest.sol");
+const ERC721LazyMintTransferProxy = artifacts.require("ERC721LazyMintTransferProxyTest.sol")
+const ERC1155LazyMintTransferProxy = artifacts.require("ERC1155LazyMintTransferProxyTest.sol")
+
+const { Order, Asset } = require("../../scripts/order.js");
+const ZERO = "0x0000000000000000000000000000000000000000";
+const { expectThrow, verifyBalanceChange } = require("@daonomic/tests-common");
+const { ETH, ERC20, ERC721, ERC1155, ORDER_DATA_V1, ORDER_DATA_V2, TO_MAKER, TO_TAKER, PROTOCOL, ROYALTY, ORIGIN, PAYOUT, CRYPTO_PUNK, COLLECTION, enc, encDataV2, id } = require("../../scripts/assets.js");
+
+contract("RaribleTransferManagerTest:doTransferTest()", accounts => {
+  let RTM;
+  let transferProxy;
+  let erc20TransferProxy;
+  let royaltiesRegistry;
+
+  const protocol = accounts[9];
+  const community = accounts[8];
+  const erc721TokenId0 = 52;
+  const erc721TokenId1 = 53;
+  const erc1155TokenId1 = 54;
+  const erc1155TokenId2 = 55;
+  const protocolCommission = 300;
+
+  before(async () => {
+    transferProxy = await TransferProxyTest.new();
+    erc20TransferProxy = await ERC20TransferProxyTest.new();
+    RTM = await RaribleTransferManagerTest.new();
+    royaltiesRegistry = await TestRoyaltiesRegistry.new();
+
+    await RTM.init____(transferProxy.address, erc20TransferProxy.address, protocolCommission, community, royaltiesRegistry.address);
+    /*
```

Comment:
Do we need this comment lines?

Reply:
I think we need to add some unit tests in /lib-bp, like in /lib-signature in the future future

Reply:
Also I don`t see any unit test/tests for V3 orders. Maybe I didn`t find it?